runtime/onert/core/include/ir/Model.h

```diff
@@ -175,8 +175,26 @@ class Model
 
 private:
   std::shared_ptr<backend::custom::IKernelBuilder> _kernel_builder;
-};
 
+public:
+  void add_metadata(const std::string &name, std::shared_ptr<const ir::Data> data)
+  {
+    _metadatas.emplace(name, data);
+  }
+
+  bool is_metadata_exist(const std::string &name) const
+  {
+    return _metadatas.find(name) != _metadatas.end();
+  }
+
+  std::shared_ptr<const ir::Data> get_metadata(const std::string name) const
+  {
+    return _metadatas.at(name);
+  }
```

Comment:
How about moving these public functions to one place on top public specifier?

Reply:
As I understood, This file(Model.h) follows this rule. 

```
public: 
   ... some interface to access the field. 
private : 
   ... filed
``` 

* `_kernel_builder` and its related interface
https://github.com/Samsung/ONE/blob/44b7707080e52b0d90a70800edf1a942371988ed/runtime/onert/core/include/ir/Model.h#L164-L177 

* `_subgraph` and its related interface
https://github.com/Samsung/ONE/blob/44b7707080e52b0d90a70800edf1a942371988ed/runtime/onert/core/include/ir/Model.h#L139-L161

Reply:
> As I understood, This file(Model.h) follows this rule.

So I tried the follow the convention as I understood.
Do you still think it is better to move the public method to the top?  
If you think so, I'd like to follow :)

Reply:
I understood that these functions were implemented in the same way as other functions in this file. Keep it as it is. :)